cmd/collectors/zapi/collector/zapi.go

```diff
@@ -169,6 +170,14 @@ func (z *Zapi) InitCache() error {
 		} else {
 			z.Logger.Trace().Msg("using default no batch-size")
 		}
+
+		z.inputParams = make(map[string]string)
+		if inputParams := z.Params.GetChildS("input_param"); inputParams != nil {
+			for _, inputParam := range inputParams.GetChildren() {
+				z.inputParams[inputParam.GetNameS()] = inputParam.GetContentS()
+			}
+			z.Logger.Debug().Msgf("using %d input params", len(z.inputParams))
```

Comment:
z.Logger.Debug().Int("len", len(z.inputParams)).Msg("using input params")

Reply:
code has been removed as not to expose new input param in ZAPI.